packages/sdk/src/utils/helpers.ts

```diff
@@ -1,24 +1,38 @@
+import { scale } from '@voltz-protocol/commons-v2';
 import md5 from 'crypto-js/md5';
-import { BigNumber, utils } from 'ethers';
-import { BaseTrade } from './types';
-import { getTokenInfo } from './constants';
+import { BigNumber } from 'ethers';
+import { RAY, WAD } from './constants';
+import { Q96 } from './math/constants';
+import { fixedRateToPrice } from './math/tickHelpers';
+
+type CreateAccountParams = {
+  ownerAddress: string;
+  productAddress: string;
+  marketId: string;
+  maturityTimestamp: number;
+  isLp: boolean;
+  priceLower?: number;
+  priceUpper?: number;
+};
 
 // would be good to turn prices into ticks to decrease chances of collision
-export async function createAccountId(
-  trade: BaseTrade,
-  isLp: boolean,
-  priceLower?: number,
-  priceUpper?: number,
-): Promise<string> {
+export async function createAccountId({
+  ownerAddress,
+  productAddress,
+  marketId,
+  maturityTimestamp,
+  isLp,
+  priceLower,
```

Comment:
do accountIds of takers also need priceLower and priceUpper or is this just a requirement for makers ?

Reply:
just for makers